module_utils/network_lsr/argument_validator.py

```diff
@@ -2216,6 +2216,14 @@ def _ipv6_is_not_configured(connection):
                     idx,
                     "ip.ipv6_disabled is not supported by initscripts.",
                 )
+            if not hasattr(Util.NM(), "SETTING_IP6_CONFIG_METHOD_DISABLED"):
+                raise ValidationError.from_connection(
+                    idx,
+                    "ip.ipv6_disabled is not supported by the running version of "
+                    "NetworkManger. But you can make the role ignore IPv6 for this "
+                    "connection by setting ip.auto6 to False and not specifying any "
+                    "IPv6 address.",
```

Comment:
It would be good to mention the version that is required to get the support.  Also, the role will not ignore IPv6 but NetworkMananger and it seems to me valid to assume that user do not specify IPv6 addresses if they want to disable this, so the main fact is that auto6 needs to be disabled (please check/adjust my proposal to describe the effects properly). Also, it would be good to mention what will still be different on the system compared to setting ipv6_disabled to True (not in the NM config but the effect on the actual system/the kernel IP configuration). Maybe this actually belongs to the README and this should mainly be a pointer to that one.

```suggestion
                    "ip.ipv6_disabled is not supported by the running version of "
                    "NetworkManger, it requires at least version ... . To disable IPv6 auto configuration, set ip.auto6 to False. Then NetworkManager will ignore IPv6 for this profile. This will still ...",
```